src/cascadia/TerminalApp/TerminalPage.cpp

```diff
@@ -2064,6 +2085,7 @@ namespace winrt::TerminalApp::implementation
 
         Command::ExpandCommands(copyOfCommands,
                                 profiles,
+                                { &sortedSchemes[0], sortedSchemes.size() },
```

Comment:
this won't pass SA. Is there an easier way to make a span

Reply:
d'oh